### DISCUSSION TITLE
How to remove a script that was injected by the Meta plugin?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#12355] [General - Components / Directives / etc] [answered]
I have injected a script tag using the Meta plugin like so:

```
<script setup land="ts">
import { useMeta } from 'quasar';
const metaData = {
  script: {
    ldJson: {
      type: 'text/javascript',
      src: 'https://desk.zoho.com.au/portal/api/web/inapp/9039000000095538?orgId=7001723021',
      nonce: '{place_your_nonce_value_here}',
      defer: true,
    },
  },
};
useMeta(metaData);
</script>
```

This 3rd party script from Zoho Desk injects an `aside` element into the page. It's a floating button users can click on to get support.

But when I navigate away from the page where this was setup, the floating button is still visible. The script tag and it's HTML stays, even on components where I have not added the Meta plugin script.

So how do I remove the script and it's associated HTML after navigating away from the page?

**Comment by rstoenescu** [comment, marked as answer]
Once a script has been injected the browser executes it. When you navigate away and Meta detects the script should no longer be rendered it removes it. But there is no way to revert the scripts runtime actions. This is how web works...
